data/tabular/flashpoint/meta.yaml

```diff
@@ -19,6 +19,7 @@ targets:
 identifiers:
     - id: SMILES
       type: SMILES
+      description: SMILES
```

Comment:
shouldn't we just have this as default for `SMILES`? I.e., instead of no going through it and adding it every time we, could simply use "SMILES" in those cases?

Reply:
yeah, not sure if we want to extend that?
I'm not sure why we set it up like that.
But in the end like that it is consistent with the other possible options.

Reply:
i had the description field originally there to also allow for more "unusual" identifiers, but let's not change a working system...